genome_sampler/sample_longitudinal.py

```diff
@@ -20,12 +21,15 @@ def _sampler(df):
 
 
 def sample_longitudinal(dates: qiime2.CategoricalMetadataColumn,
+                        context_seqs: DNAFASTAFormat,
```

Comment:
should this be optional?

Reply:
That's a good idea. I guess the only issue is that if a user doesn't realize they need to provide it, it'll result in an annoying failure later in the workflow. But Snakemake includes it, and this step is fast. Yeah, let's make it optional. I'll push that up.